### PR TITLE
Add Internationalization section

### PR DESCRIPTION
**Pull request by msporny**
This PR is an attempt to address issue #211 by adding an Internationalization Considerations section that covers:

1. How to express i18n metadata in both JSON and JSON-LD by pointing to other NOTEs and RECs that define how to do so, and 
2. Examples on how to express i18n information in both JSON and JSON-LD.

As covered [during the call today](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-10-17-vcwg-special#section1-1), we will merge this PR faster than the typical 7 day window after we have multiple reviews. I plan to merge it by Thursday if there are no objections and at least two reviews.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-integrity/pull/213.html" title="Last updated on Oct 21, 2023, 12:01 AM UTC (57f0e78)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-integrity/213/a6377ee...57f0e78.html" title="Last updated on Oct 21, 2023, 12:01 AM UTC (57f0e78)">Diff</a>

### CODE REVIEW

**Review by iherman — APPROVED** [review]
I am fine with merging this PR. However, on longer term, I think the issue I raised for the VCDM (https://github.com/w3c/vc-data-model/issues/1313) is relevant here as well. But possible changes on the examples are purely editorial, not affecting any normative section, so no reason to hold up the merge.

**Comment by msporny** [comment]
> I think the issue I raised for the VCDM ([w3c/vc-data-model#1313](https://github.com/w3c/vc-data-model/issues/1313)) is relevant here as well.

Yes, agreed, we'll need a better example that makes using `direction` mandatory for proper display of the text.

**Review by dlongley — APPROVED** [review]

**Review by longpd — APPROVED** [review]

**Review by Wind4Greg — APPROVED** [review]

File: index.html
Comment (aphillips) [inline index.html]: I think the change you made above (to replace the string "hello world" with a data representation) addresses our (pending) comment. 

I'm not sure I would add an I18N Considerations section. (Generally I18N recommends writing a section like this only when it goes beyond "eat your vegetables".) While there isn't really anything wrong with the text here, it's somewhat orthogonal to this specific spec's task. Implementers will need to read the other VC specs anyway, where they will find similar guidance in a more directly-relevant context. 

So... unless you disagree, I would just drop this?

**Review by aphillips — CHANGES_REQUESTED** [review]

File: index.html
Comment (TallTed) [inline index.html]:
```suggestion
There is currently no standard way to express language and/or text direction
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Small tweak

File: index.html
Comment (msporny) [reply index.html]: Ok, I'll drop the section before merging this. I was attempting to give [[STRING-META]] some airtime along w/ pointing folks to relevant i18n sections elsewhere. That said, seems like there is support to drop it, so I will.

Comment (msporny) [reply index.html]: Done in https://github.com/w3c/vc-data-integrity/pull/213/commits/57f0e78a2a8b5713e53c4af5307a3c60a823350f

File: index.html
Comment (msporny) [reply index.html]: This section was removed as requested in https://github.com/w3c/vc-data-integrity/pull/213#discussion_r1364183506, so the change suggestion wasn't applied.

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**
